interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +840,23 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+
+<?php if(isset($_GLOBALS['validate_pages']))  :?>
+    <?php foreach($_GLOBALS['validate_pages'] as $validation) :?>
+        <?php if($validation['page_name']==basename(__FILE__)):?>
+            <?php include_once("$srcdir/validation/validation_script.js.php");?>
+            <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/jquery-1.9.1.min.js"></script>
```

Comment:
I have moved the jquery to the top of the page.

This place is the best place for validation script include . this way we ensure that validate functoin is the validate.js function like in history_full.php otherwise the function validate() { in line 594 takes place in the code.

I can move it to line 607 but it will break out of context I think.